src/components/UI/Card.svelte

```diff
@@ -0,0 +1,125 @@
+<script>
+  import { _ } from 'svelte-i18n';
+  import Text from './Text.svelte';
+  import Icon from './Icon.svelte';
+  import Image from './Image.svelte';
+  import { calendarIcon, clockIcon, arrowIcon } from '@/images/icons';
+
+  export let languageAbbreviation = 'EN';
+  export let title = '';
+  export let src = '';
+  export let date = '';
+  export let time = '';
+  export let href = '';
+  export let group = null;
+
+  const openLinkInNewTab = () => {
+    window.open(href, '_blank');
+  };
+</script>
+
+<div class="card" on:click={openLinkInNewTab}>
```

Comment:
For this to be better, semantic HTML, `article` would be a better suited top-level component for `div`

Additionally, to make this component accessible, you can't just give a `div` a click handler. People using screen readers or other assistive technology will not be able to use it.
We have a utility class called `.button-container` which is basically an button with all its styles set to inherit whatever else the styles the element has.

I think the correct solution here is a `button` element with the `button-container` class, that wraps an `article` with the `card` class. See `ConversationCard`, `Collapsible` and  `Hamburger`, among others, for examples.